master/internal/resourcemanagers/agent_resource_manager.go

```diff
@@ -500,3 +508,39 @@ func (a *agentResourceManager) createResourcePoolSummary(
 
 	return resp, nil
 }
+
+func (a *agentResourceManager) fetchAvgQueuedTime(pool string) (
+	[]*jobv1.AggregateQueueStats, error,
+) {
+	aggregates := make([]model.ResourceAggregates, 0)
```

Comment:
somewhat of a style preference, but unless you want to preallocate some known capacity (e.g. `make([]model.ResourceAggregates, 0, len(someResourceMap))`, I'd say curly bracket initialization looks nicer
```suggestion
	aggregates := []model.ResourceAggregates{}
```

Reply:
Fixed